content/src/logic/deployments.ts

```diff
@@ -52,10 +53,14 @@ export async function retryFailedDeploymentExecution(
         )
       } catch (error) {
         // it failed again, override failed deployment error description
-        const errorDescription = error.message
-        components.failedDeploymentsCache.reportFailure({ ...failedDeployment, errorDescription })
+        const errorDescription = error.message + ''
 
-        logger.info(`Failed to fix deployment of entity`, { entityId, entityType })
+        // TODO [mendez] this condition has no test coverage
+        if (!errorDescription.startsWith(IGNORING_FIX_ERROR)) {
```

Comment:
The ignore of a failed deployment should correspond to fix it, not a fail: for example: if a failed deployment needs to be ignored that's because there's a new deployment for the same pointer correctly deployed